singer-io/tap-activecampaign
# TDL-16481 Update schema

# Description of change
- Updated schema for following stream
  - deal_stages
  - contact_custom_fields
  - conversions
  - templets
  - contact_list
  - brandings
  - forms
  - users
  - tasks
  - contact
  - campaign_links
  - deal_activities
- Updated property `params` of automation_blocks to support empty array data type.
- Added all_fields integration test case.
- Removed unnecessary files(test case for snapchat_ads) under tests directory.

# Manual QA steps
 - Run discover mode and verify that fields replicate as expected in catalog
 - Run sync mode and verify that all fields replicate in output.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
